analysers/experiments/IO.py

```diff
@@ -57,6 +57,9 @@ def f(r):
             .filter(lambda a: a["device"] == dev) \
             .map(f) \
             .cache()
+            
+    if (dataRDD.first())[0] is None:
```

Comment:
@VincenzoFerme If we don't have values for one of the operations it shows as null (None in Python), so I can't compute metrics on them.